pages/linux/virt-viewer.md

```diff
@@ -0,0 +1,25 @@
+# virt-viewer
+
+> Minimal graphical interface for a virtual machine (VM).
+> NOTE: 'domain' refers to the name, UUID or ID for the existing VMs (See: tldr virsh).
+> More information: <https://gitlab.com/virt-viewer/virt-viewer/-/blob/master/man/virt-viewer.pod>.
+
+- Launch virt-viewer with prompt to select running virtual machines:
+
+`virt-viewer`
+
+- Launch virt-viewer for a specific virtual machine by ID, UUID or name:
+
+`virt-viewer {{domain}}`
+
+- Wait for a virtual machine to start and automatically reconnect if it shutdown and restarts:
+
+`virt-viewer --reconnect --wait {{domain}}`
+
+- Connect to remote virtual machine over TLS:
+
+`virt-viewer --connect xen//{{url}} {{domain}}`
```

Comment:
```suggestion
- Connect to a specific remote virtual machine over TLS:

`virt-viewer --connect "xen//{{url}}" "{{domain}}"`
```

- Optional fix: `a specific` may sound better.
- Optional fix: `xen//{{url}}` and `{{domain}}` can be quoted if they can contain some special characters to POSIX-like compatible shell.